projects/plugins/jetpack/modules/search/instant-search/components/overlay.jsx

```diff
@@ -44,7 +44,7 @@ const Overlay = props => {
 			role="dialog"
 		>
 			<h1 id="jetpack-instant-search__overlay-title" className="screen-reader-text">
-				{ __( 'Search results', 'jetpack' ) }
+				{ __( 'Jetpack Search Results', 'jetpack' ) }
```

Comment:
Is it relevant to the user that Jetpack Search produced the results?

I think I'd be tempted to stick with the original text.

Reply:
Do you see any value in giving the user additional branding context here? Visually, it's easy to tell that Jetpack Search produced the overlay. Using a screen reader, however, you'd need to drill all the way down to the bottom of the sidebar, like so:

<img width="1055" alt="Screen Shot 2022-01-10 at 3 42 02 PM" src="https://user-images.githubusercontent.com/4044428/148850443-248ea263-8479-403e-9f49-8ee675e7be01.png">

Reply:
I reckon keeping the title brief outweighs branding in this case 👍